README.md

```diff
@@ -799,16 +530,178 @@ a:hover {
 /* Basic one-line comment */
 ```
 
-<a name="classname-conventions"></a>
-## Classname Conventions
 
-<a name="use-smacss-prefixes-to-distinguish-classes-with-different-roles"></a>
-### Use SMACSS prefixes to distinguish classes with different roles.
 
-**SMACSS** (Scalable and Modular Architecture for CSS) advocates grouping different types
-of classes based on their function. The following is a slightly modified version:
 
-All classnames should have a prefix. Prefixes help insulate new classnames from legacy classnames. The different prefixes are usually a single letter, and are explained in the following section. 
+
+
+
+<a name="object-oriented-css-oocss"></a>
+## Object-Oriented CSS (OOCSS)
+
+OOCSS advocates practices which are beneficial to flexibility, code reuse, maintenance, and performance.
+
+
+
+<a name="composition-with-multiple-classes"></a>
+### Composition with Multiple Classes
+
+One of the core ideas of Object-Oriented CSS is using multiple classnames on a single HTML element to synthesize new components from small, modular, focused styles.
+
+Order classnames in a class attribute by order of "inheritance" -- order in the class attribute does not matter functionally, but helps make the inheritance chain more apparent. 
+
+Separate multiple classes with TWO spaces for readability.
+
+```html
+<!-- Good -->
+<div class="c-tutorial-nav  c-tutorial-nav--alt  u-pull-left">
+
+<!-- Bad - don't use single space -->
+<div class="c-tutorial-nav c-tutorial-nav--alt u-pull-left">
+
+<!-- Bad - not ordered by inheritance: "alt" extends the base. -->
+<div class="c-tutorial-nav--alt  c-tutorial-nav  u-pull-left">
+```
+
+The classes should follow this order:
+
+```html
+<div class="[block|element]  [modifier  [modifier]]  [state]  [utility]  [javascript hooks]  [test hooks]">
+```
+
+
+
+
+
```

Comment:
Still seeing it here, and I remembered to refresh my tab before reviewing & checked that I wasn't on so weird revision number.